src/ex1/script.js

```diff
@@ -0,0 +1,122 @@
+const addBtn = document.querySelectorAll('.add-task-btn');
+const clearAllBtn = document.querySelectorAll('.clear-all-btn');
+const addTaskField = document.querySelectorAll('.add-task-field');
```

Comment:
i would either add [0] to all of them instead of doing it on each and every one of the usages, or just use querySelector, preferably if you expect to have only one element from this kind, you can use id's instead of classes